kie-remote/kie-remote-common/src/test/java/org/kie/remote/common/rest/KieRemoteHttpRequestTest.java

```diff
@@ -859,7 +855,7 @@ public void handle( Request request, HttpServletResponse response ) {
             }
         };
         Map<String, List<String>> headers = newRequest(url).get().response().headers();
-        assertEquals(headers.size(), 5);
+        assertEquals(headers.size(), 6);
```

Comment:
Jetty 9.x includes one more default header for some reason, but I forgot which one :) I vaguely remember it was a timestamp of the request (or response?), but can look it up to be sure.